### PR TITLE
New Mentor: MindLabor

### PR DESCRIPTION
**Pull request by MindLabor**

### CODE REVIEW

File: mentors/java/m.json [hunk 1]
```diff
@@ -63,6 +63,14 @@
   "avatar_url": null,
   "bio": "Mostly a Java developer, interested in solving problems and always looking for ways to learn new things. I'm also into cross-country skiing!"
 },
+{
+  "github_username": "MindLabor",
+  "name": "Samuel Braun",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "My first programming language was Java. At the beginning it was quite challenging to write some code that does actually make sense. After a few days however, it became more and more easier and fun to mess around with."
```
Comment (rpottsoh) [inline mentors/java/m.json]: Might want `easy` instead of `easier`.

Comment (MindLabor) [reply mentors/java/m.json]: Yeah, 'easy' makes more sense. I'm from Germany and English isn't my native language, so....

Comment (rpottsoh) [reply mentors/java/m.json]: ....or you could remove `more and more`...

Comment (MindLabor) [reply mentors/java/m.json]: I don't know. It's up to you. You're the native speaker. I am sure you will choose wisely ;)

Comment (rpottsoh) [reply mentors/java/m.json]: Either suggestion is fine.  However, I cannot make the change.  I can only approve and merge.

Comment (rpottsoh) [inline mentors/java/m.json]:
```suggestion
  "bio": "My first programming language was Java. At the beginning it was quite challenging to write some code that does actually make sense. After a few days however, it became easier and fun to mess around with."
```

**Pull request merged**